src/less/tooltip/ds6/tooltip.less

```diff
@@ -0,0 +1,404 @@
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+@import "tooltip-base.less";
+
+/* Default style */
+.tooltip,
+.infotip,
+.tourtip {
+    display: inline-block;
+    position: relative;
+
+    &__window {
+        position: absolute;
+        z-index: 1;
+
+        &::after {
+            border-style: solid;
+            border-width: 8px;
+            content: "";
+            position: absolute;
+        }
+    }
+
+    &__body {
+        margin: 16px 16px 16px 16px;
+    }
+}
+
+.tooltip {
+    &:hover {
+        .tooltip__top-arrow,
+        .tooltip__top-leftarrow,
+        .tooltip__top-rightarrow,
+        .tooltip__bottom-arrow,
+        .tooltip__bottom-leftarrow,
+        .tooltip__bottom-rightarrow,
+        .tooltip__left-arrow,
+        .tooltip__left-toparrow,
+        .tooltip__left-bottomarrow,
+        .tooltip__right-arrow,
+        .tooltip__right-toparrow,
+        .tooltip__right-bottomarrow {
+            visibility: visible;
```

Comment:
Do not use `visibility: visible;` or `visibility: hidden;`. Please refer to:

Mind pattern:
- https://ebay.gitbooks.io/mindpatterns/content/disclosure/tooltip.html

Previous DS4 tooltip:
- https://github.com/eBay/skin/blob/master/src/less/notice/ds4/notice-base.less
- https://github.com/eBay/skin/blob/master/src/less/notice/ds4/notice-tooltip.less